src/lis3dh-motion-detection.cpp

```diff
@@ -191,7 +191,64 @@ status_t LIS3DH::writeRegister(uint8_t offset, uint8_t dataToWrite) {
     returnError = IMU_HW_ERROR;
   }
 
+	_DEBBUG("Write register 0x", offset, " = ", dataToWrite);
 	return returnError;
+
+	delay(100);             // some users reported hangs without delay.
+}
+
+// Read tap (aka: 'click')
+uint8_t LIS3DH::readTap(){
```

Comment:
Tap? Do you refer to Click? I would prefer to use the datasheet nomenclature, i.e Click.

Reply:
Yes, as stated. No problem, I can change it.